### DISCUSSION TITLE
Alignment with VSCode Copilot

### DISCUSSION THREAD

**Discussion by dgibson66** [Genuitec/Copilot4Eclipse-community#26] [Help] [answered]
Github releases new versions of the VSCode plugin and it's underlying API access code.  How does Copilot4Eclipse stay aligned with the Github release?  How much of the Github improvements do we see without a Copilot4Eclipse release?

**Comment by wayneparrott** [comment, marked as answer]
Copilot4Eclipse integrates GitHub Copilot services into the Eclipse IDE which requires custom Eclipse plugin coding.  Copilot4Eclipse has no relationship to the VSCode GitHub Copilot extension. 

While we have not published Copilot4Eclipse roadmap, our initial approach has been to reach parity with the feature set of the GitHub Copilot plugin for JetBrains IntelliJ. The IJ plugin's feature set is a subset of the VSCode Copilot extension. Beyond that we will be extending Copilot4Eclipse to provide deeper levels of coding tool integration with Copilot-based agentic features tailored to the Java enterprise community, e.g., Spring/SpringBoot. 

Watch for new interesting features to appear soon in upcoming Copilot4Eclipse updates.
